app/jobs/email_privacy_policy_update_job.rb

```diff
@@ -0,0 +1,43 @@
+class EmailPrivacyPolicyUpdateJob < ApplicationJob
+  def perform(petition)
+    terminating = false
+
+    worker = trap("TERM") do
+      terminating = true
+      worker.call
+    end
+
+    Appsignal.without_instrumentation do
+      petition.signatures.find_each do |signature|
```

Comment:
Probably want to scope this to validated signatures - don't really want to send emails to signatures that have been invalided or marked as fraudulent and pending signatures are likely to bounce which could put our SES account in jeopardy if we get a lot in a short period.